project/Build.scala

```diff
@@ -321,7 +325,7 @@ object Build extends sbt.Build {
   )
 
   val commonToolsSettings = (
-      commonSettings ++ publishSettings
+      commonSettings ++ publishSettings ++ fatalWarningsSettings
```

Comment:
I believe the tools have warnings (pattern matching not being able to prove reachability or some other nonsense)